### ISSUE TITLE
Error: Cannot find module 'react/lib/React' on React version ^16.2.0

### ISSUE THREAD

**Issue by epfremmer** [pzavolinsky/react-cucumber#2]
It looks like react-unit has the version of `react-test-renderer` locked at 15.5.4 which may be incompatible with react version 16.x judging by this stack overflow post: https://stackoverflow.com/questions/46692073/jest-fails-cannot-find-module-react-lib-react-from-reactshallowrenderer-js-a

I did do some mild testing and it looks like upgrading react-test-renderer to version 16.x can resolve this issue.

```
Error: Cannot find module 'react/lib/React'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (<project_root>/node_modules/react-unit/node_modules/react-test-renderer/lib/shallow/ReactShallowRenderer.js:18:13)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (<project_root>/node_modules/react-unit/node_modules/react-test-renderer/shallow.js:3:18)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
```

Let me know if you have any other questions about my specific system or configuration to help debug this.

p.s. this library looks really cool 👍 looking forward to using it on my projects!

**Comment by epfremmer** [comment]
Created a PR that i think should fix the issue: pzavolinsky/react-unit#28

**Comment by pzavolinsky** [comment]
Hi @epfremmer thanks for this contrib!

I've merged your pzavolinsky/react-unit#28 PR and published a new version of `react-cucumber` with updated deps!

Thanks again,
Cheers!

**Comment by epfremmer** [comment]
@pzavolinsky happy to help!
